recipe/common.php

```diff
@@ -69,12 +69,19 @@
 
 
 /**
- * Return current release path.
+ * Custom paths.
  */
-set('current_path', function () {
-    $link = run("readlink {{deploy_path}}/current")->toString();
-    return substr($link, 0, 1) === '/' ? $link : get('deploy_path') . '/' . $link;
-});
+set('current_dir', 'current');
+set('release_dir', 'release');
+set('releases_dir', 'releases');
+set('shared_dir', 'shared');
+set('dep_dir', '.dep');
+
+set('current_path', '{{deploy_path}}/{{current_dir}}');
```

Comment:
Why using a symlink will lead to bugs on the code? I am letting the OS to resolve the symlink instead of forcing a readlink.